extensions/ql-vscode/.storybook/preview.js

```diff
@@ -0,0 +1,38 @@
+import { themes } from '@storybook/theming';
+import { action } from '@storybook/addon-actions';
+
+// Allow all stories/components to use Codicons
+import '@vscode/codicons/dist/codicon.css';
```

Comment:
Can you remind me why this isn't just part of our webpack bundle?

Reply:
This is part of our normal Webpack bundle, but it's only imported by the `webview.ts` file, which we're not explicitly importing in Storybook. Storybook uses a different Webpack config, so we need to explicitly import it.

Reply:
Ok thanks. I think there's a way to tell webpack to always include it without having to import but that's a problem for another day :D

Reply:
There's a way to specify the entrypoint for Webpack so that it always imports this file, but that won't really solve our problem since we're using two different Webpack configs (one for the extension and one for Storybook). So, we would still need to import the Codicons twice.